### PR TITLE
Us10 new availability types

### PR DESCRIPTION
**Pull request by drauf**
Tests were fixed in #22, so they'll work after merging from master.

### CODE REVIEW

File: src/main/resources/assets/html/partials/interviewer/tdp-interviewer-home.html [hunk 1]
```diff
@@ -4,32 +4,41 @@
...
         <tbody>
-            <tr ng-repeat="slotsForDay in slotsForWeek">
-                <td class="slotTime">{{slotTimes[$index]}}</td>
-                <td ng-repeat="slot in slotsForDay track by $index" ng-class="{'available-slot': slot.available}" ng-click="slot.available = !slot.available">
-                </td>
-            </tr>
+        <tr ng-repeat="slotsForDay in slotsForWeek">
+            <td class="slotTime">{{slotTimes[$index]}}</td>
+            <td ng-repeat="slot in slotsForDay track by $index"
+                ng-class="{'available-slot': slot.type.id == 1, 'unavailable-slot': slot.type.id == 4, 'maybe-available-slot': slot.type.id == 5}"
```
Comment (RogowskaMalgorzata) [inline src/main/resources/assets/html/partials/interviewer/tdp-interviewer-home.html]: compare with === not ==

Comment (mateuszs-kainos) [inline src/main/resources/assets/html/partials/interviewer/tdp-interviewer-home.html]: It doesn't work with ===.

File: src/main/resources/assets/js/application/interviewer/controllers/tdprInterviewerHomeController.js [hunk 2]
```diff
@@ -1,51 +1,48 @@
...
+                var endTime = response.data[i].endTime.slice(0, 5);
                 $scope.slotTimes.push(startTime + "-" + endTime);
             }
-    });
+        });
 
         $scope.startDate;
         $scope.endDate;
-
+        $scope.currentType = '1'; // 1 - available, 2 - full, 3 - init, 4 - unavailable, 5 - maybe
```
Comment (wojciechh-kainos) [inline src/main/resources/assets/js/application/interviewer/controllers/tdprInterviewerHomeController.js]: We should make this into an enum - an angular constant object which we could inject - like tdprAvailabilityTypes
{available: 'available', full: 'full', init: 'init', unavailable: 'unavailable', maybe: 'maybe'} - do we need to use these numbers?

File: src/main/resources/assets/html/partials/interviewer/tdp-interviewer-home.html [hunk 3]
```diff
@@ -4,32 +4,46 @@
...
         <tbody>
-            <tr ng-repeat="slotsForDay in slotsForWeek">
-                <td class="slotTime">{{slotTimes[$index]}}</td>
-                <td ng-repeat="slot in slotsForDay track by $index" ng-class="{'available-slot': slot.available}" ng-click="slot.available = !slot.available">
-                </td>
-            </tr>
+        <tr ng-repeat="slotsForDay in slotsForWeek">
+            <td class="slotTime">{{slotTimes[$index]}}</td>
+            <td ng-repeat="slot in slotsForDay track by $index"
+                ng-class="{'{{AvailabilityEnum.available.className}}': slot.type === AvailabilityEnum.available.id,
```
Comment (wojciechh-kainos) [inline src/main/resources/assets/html/partials/interviewer/tdp-interviewer-home.html]: Can't we do class={{AvailabilityEnum[slot.type].class}} ?

File: src/main/resources/assets/js/test/interviewerHomeControllerTest.js [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+
+            it('should store values received from service in slotsForWeek array', function () {
+                getSlotsDeferred.resolve({
+                    data: [{slot: {id: 1}, type: {id: 61}, slotsDate: getDayOfTheWeek(new Date(), 1)}, // 1st slot, tuesday
+                        {slot: {id: 4}, type: {id: 62}, slotsDate: getDayOfTheWeek(new Date(), 4)}, // 4th slot, friday
+                        {slot: {id: 6}, type: {id: 63}, slotsDate: getDayOfTheWeek(new Date(), 0)}] // 6th slot, monday
+                });
+
+                $scope.getSlots();
+                $scope.$apply();
```
Comment (wojciechh-kainos) [inline src/main/resources/assets/js/test/interviewerHomeControllerTest.js:69]: why do we need apply here?

File: src/main/resources/migrations.xml [hunk 5]
```diff
@@ -12,4 +12,6 @@
     <include file="changelogs/initialChangeColumnDayNameInSlots.xml"/>
     <include file="changelogs/initialPopulatePersonsTable.xml"/>
     <include file="changelogs/initialPopulateSlotsTable.xml"/>
+    <include file="changelogs/initialPopulateAvailabilityTypes.xml"/>
+
 </databaseChangeLog>
```
Comment (wojciechh-kainos) [inline src/main/resources/migrations.xml:17]: Code review done

File: src/main/resources/assets/js/test/interviewerHomeControllerTest.js (hunk 4 above)
Comment (drauf) [inline src/main/resources/assets/js/test/interviewerHomeControllerTest.js:69]: getSlotsDeferred.resolve() will only run after apply

**Pull request merged**